docs/concepts/hashing.md

```diff
@@ -39,8 +35,11 @@ For example, the SHA-256 hash of "Hello world" from above can be represented as
 ```
 mtwirsqawjuoloq2gvtyug2tc3jbf5htm2zeo4rsknfiv3fdp46a
 ```
```

Comment:
Add empty line after codeblocks. Also, try and give each codeblock syntax highlighting. If there isn't a syntax we can use, like in this case, use `plaintext` intstead.

````suggestion
```plaintext

````